apps/docs/docs/Guide/learning-the-workflow/deploying.md

```diff
@@ -0,0 +1,79 @@
+---
+sidebar_position: 4
+---
+
+# Deploying
+
+Disploy allows you to deploy your Discord application to your target of choice expressed in the `disploy.json` file. We currently only support deploying to [Cloudflare Workers](https://workers.cloudflare.com/).
+
+## Deploying to Cloudflare Workers
+
+To deploy to Cloudflare Workers, you need to have a Cloudflare account, a Cloudflare Workers subscription and [wrangler](https://developers.cloudflare.com/workers/wrangler/) installed and configured.
+
+- You can sign up for a Cloudflare account [here](https://dash.cloudflare.com/sign-up).
+- Sign up for a Cloudflare Workers subscription [here](https://dash.cloudflare.com/sign-up/workers) (you can sign up for a free Workers subscription with [limited usage](https://developers.cloudflare.com/workers/platform/pricing), 100,000 requests per day).
+- Install wrangler via npm by running the following command:
+
+```bash
+npm install -g wrangler
+```
+
+- Configure wrangler by running the following command:
+
+```bash
+wrangler login
+```
+
+Once you have a Cloudflare account and a Cloudflare Workers subscription, you can deploy your Discord application by running the following command:
+
+```bash
+$ disploy deploy
+```
+
+This will deploy your Discord application to a Cloudflare Worker defined in the `disploy.json` file.
+
+```json
+{
+	"prebuild": "yarn run build",
+	"root": "dist",
+	"target": {
+		"type": "cloudflare",
+		"name": "cf-example" // The name of your Cloudflare Worker
+	}
+}
+```
+
+### Configuring environment variables
+
+You will need to configure the following environment variables in your Cloudflare Worker:
+
+- `CLIENT_ID`: The client ID of your Discord application.
+- `PUBLIC_KEY`: The public key of your Discord application.
+- `TOKEN`: The token of your Discord application.
+
+You can find the client ID, public key and token of your Discord application in the [Discord Developer Portal](https://discord.com/developers/applications).
+
+<details>
+    <summary> Click to see how to find the client ID, public key and token of your Discord application </summary>
+
+![](https://cdn.discordapp.com/attachments/1038456602610651196/1038456753672695848/id-and-pubkey.png)
```

Comment:
Images should be uploaded into the repo and linked there, instead of using a third-party CDN.

Reply:
I tried that originally and they wouldn’t push because they were too big, anyways I don’t think we should have images in the git history anyways since this is a monorepo and it will make the git repository huge and slow.